api/common/authentication/authenticateToken.js

```diff
@@ -0,0 +1,11 @@
+import dt from '../../../config/dotenvInit.js';
+import jwt from 'jsonwebtoken';
+
+export default async function authenticateToken(token) {
+    return jwt.verify(token, process.env.SECRET, (err, user) => {
```

Comment:
1. Please move use of jwt directly to lib/middleware/authenticateToken.js. In such case you could remove api/common/authentication/authenticateToken.js
(read comments for api/common/authentication/controller.js to know how to create lib/middleware/authenticateToken.js)

2. Get 'SECRET' var from config/index.js not from process.env